addons/binding/org.openhab.binding.smartmeter/README.md

```diff
@@ -105,17 +105,18 @@ Any help/contribution is appreciated!
 
 The binding has been successfully tested with below hardware configuration:
 
-SML PUSH mode:
+### SML PUSH mode:
 
 - EMH EDL300 meter connected the IR-Reader USB from open hardware project in volkszaehler
 - EMH eHZ-IW8E2A
 - ISKRA MT681
 - EMH eHZ-K
 
-IEC 62056-21 Mode C:
+### IEC 62056-21 Mode C:
 
 - Apator EC3 with IR-Reader from volkszaehler
+- Landis+Gyr E650 with IR-Reader from volkszaehler
 
-IEC 62056-21 Mode D:
+### IEC 62056-21 Mode D:
```

Comment:
I don't mind this is changed to a header. But then the trailing colon needs to be removed according to the style guide we use for markdown files:

http://www.cirosantilli.com/markdown-style-guide/#punctuation-at-the-end-of-headers

Reply:
Thanks & sorry, I missed the colons. Fixed now.